dapr/aio/clients/grpc/client.py

```diff
@@ -1103,6 +1110,278 @@ async def unlock(
+                                       properties=resp.properties)
+        except grpc.aio.AioRpcError as err:
+            raise DaprInternalError(err.details())
+
+    async def terminate_workflow(
+            self,
+            instance_id: str,
+            workflow_component: str) -> DaprResponse:
+        """Terminates a workflow.
+
+            Args:
+                instance_id (str): the ID of the workflow instance, e.g.
+                                    `order_processing_workflow-103784`.
+                workflow_component (str): the name of the workflow component
+                                    that will run the workflow. e.g. `dapr`.
+
+            Returns:
+                :class:`DaprResponse` gRPC metadata returned from callee
+
+        """
+        # Warnings and input validation
+        warn('The Workflow API is an Alpha version and is subject to change.',
+             UserWarning, stacklevel=2)
+        validateNotBlankString(instance_id=instance_id,
+                               workflow_component=workflow_component)
+        # Actual terminate workflow invocation
+        req = api_v1.TerminateWorkflowRequest(
+            instance_id=instance_id,
+            workflow_component=workflow_component)
+
+        try:
+            _, call = self._stub.TerminateWorkflowAlpha1.with_call(req)
+            return DaprResponse(
+                headers=call.initial_metadata())
+        except grpc.aio.AioRpcError as err:
+            raise DaprInternalError(err.details())
+
+    async def raise_workflow_event(
+            self,
+            instance_id: str,
+            workflow_component: str,
+            event_name: str,
+            event_data: Union[Any, bytes, None] = None) -> DaprResponse:
+        """Raises an event on a workflow.
+
+            Args:
+                instance_id (str): the ID of the workflow instance,
+                                    e.g. `order_processing_workflow-103784`.
+                workflow_component (str): the name of the workflow component
+                                    that will run the workflow. e.g. `dapr`.
+                event_name (str): the name of the event to be raised on
+                                    the workflow.
+                event_data (Union[Any, bytes, None]): the input to the event.
+
+            Returns:
+                :class:`DaprResponse` gRPC metadata returned from callee
+        """
+        # Warnings and input validation
+        warn('The Workflow API is an Alpha version and is subject to change.',
+             UserWarning, stacklevel=2)
+        validateNotBlankString(instance_id=instance_id,
+                               workflow_component=workflow_component,
+                               event_name=event_name)
+        if isinstance(input, bytes):
+            encoded_data = event_data
+        else:
+            if event_data is not None:
```

Comment:
Why not combine the if else block like you did in start_workflow?

```python
encoded_data = json.dumps(input).encode("utf-8") if input is not None else bytes([])
```

Reply:
Add the try catch as above, but change the error message to `raise_workflow_event` and `event_data`

Reply:
The line was too long for the linter so I split it into multiple